torch/onnx/symbolic_opset9.py

```diff
@@ -462,13 +462,10 @@ def view(g, self, size):
     if sym_help._is_value(size):
         shape = size
     else:
-        if self.isCompleteTensor():
-            self_sizes = self.type().sizes()
-            if self_sizes and len(size) == 2 and self_sizes[0] == size[0]:
-                return g.op("Flatten", self, axis_i=1)
         shape = g.op("Constant", value_t=torch.LongTensor(size))
```

Comment:
I'm wondering if we need this step anymore. This seems due to that we are parsing `size` to get const by `is`, and now putting it back to tensor through `Constant` node. Maybe we can remove all these, and just directly return `g.op("Reshape", self, size)`